requirements/requirements.md

```diff
@@ -1,20 +1,21 @@
 # Requirements
 
-A collection of requirements and scenarios, framing the scope of the Notary v2 project.
+A collection of requirements and scenarios, framing the scope of the Notary project.
 
 ## TOC
 
+<!-- no toc -->
 - [Goals](#goals)
 - [Non Goals](#non-goals)
 - [Scenarios](./scenarios.md)
-- [Threat Model](./threatmodel.md)
-- [Key Stake Holders & Contributors](#key-stake-holders--contributors)
+- [Key Stake Holders \& Contributors](#key-stake-holders--contributors)
+- [Contributing \& Conversations](#contributing--conversations)
 - [Definitions & Terms](./definitions-terms.md)
 - [Contributing & Conversations](#contributing--conversations)
 
 ## Goals
 
-Notary v2 aims to address the learnings and gaps of v1, while prioritizing a set of goals and [scenarios](./scenarios.md).
+Notary Project aims to address the learnings and gaps of legacy Notary, while prioritizing a set of goals and [scenarios](./scenarios.md).
```

Comment:
Is _"legacy Notary"_ the way to refer to https://github.com/notaryproject/notary now?

Reply:
Yes, it referred to https://github.com/notaryproject/notary now. I will update it to `gaps of previous version` based on the feedback from Steve. `legacy` is not a good wording. cc @toddysm for comments.

Reply:
As clarified https://github.com/notaryproject/.github/issues/35, it will be updated to `TUF-based implementation`